hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/ChunkKeyHandler.java

```diff
@@ -80,13 +79,13 @@ protected void execute(OzoneClient client, OzoneAddress address)
         XceiverClientManager xceiverClientManager = containerOperationClient.getXceiverClientManager()) {
       OzoneManagerProtocol ozoneManagerClient = client.getObjectStore().getClientProxy().getOzoneManagerClient();
       address.ensureKeyAddress();
-      JsonElement element;
-      JsonObject result = new JsonObject();
+      ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Have you considered using `JsonUtil`, as mentioned in https://github.com/apache/ozone/pull/6147#pullrequestreview-1941518871?

Reply:
Thanks for reminding!
I have implemented the changes!!